src/AmbientSounds/ViewModels/SettingsViewModel.cs

```diff
@@ -8,10 +8,30 @@ public class SettingsViewModel
     {
         private readonly IUserSettings _userSettings;
 
+        private readonly string _setting;
+
         public SettingsViewModel(IUserSettings userSettings)
         {
             Guard.IsNotNull(userSettings, nameof(userSettings));
             _userSettings = userSettings;
+            _setting = _userSettings.Get<string>("themeSetting");
+            if (_setting != null)
+            {
+                switch (_setting)
+                {
+                    case "default":
+                        DefaultRadioIsChecked = true;
+                        break;
+                    case "light":
+                        LightRadioIsChecked = true;
+                        break;
+                    case "dark":
+                        DarkRadioIsChecked = true;
+                        break;
+                    default:
+                        break;
+                }
+            }
```

Comment:
nitpick: can you move this to a private method called InitializeTheme? This way, the constructor can stay concise.